deployments/mainnet/weth/configuration.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Compound WETH",
+  "symbol": "cWETHv3",
+  "baseToken": "WETH",
+  "baseTokenAddress": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+  "baseTokenPriceFeed": "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419",
+  "borrowMin": "100e6",
```

Comment:
bumping this so we don't forget: `borrowMin`, `targetReserves`, and `baseMinForRewards` are all 6 decimal values, not 18 decimal values

Reply:
Here's what I'm thinking for the tracking index values:
```
trackingIndexScale: 1e12 or 1e15
baseMinForRewards: 1_000e18 (is this too high? similar to USDC market, this would be around $1mn)
```